scripts/exec.mjs

```diff
@@ -28,20 +31,25 @@ export function execute(command, args = []) {
 
     proc.stdout.on('data', (chunk) => {
       console.log(trimNewline(chunk.toString()));
-      data += chunk.toString();
+      dataChunks.push(Buffer.from(chunk));
```

Comment:
Why not just use buffer from the get-go?
Solutions B, exposes or copy the spawnUtils from semantic monorepo tools, you'll save some yourself some headaches

Reply:
I never really familiarized myself with buffers, I didn't think they were as simple as they actually are, and I was copying part of a recipe from StackOverflow. I'm gonna use buffers directly instead of having an array of buffers.